.github/renovate.json

```diff
@@ -67,7 +67,8 @@
       ],
       "matchPackageNames": [
         "styled_components"
-      ]
+      ],
+      "enable": false
```

Comment:
how about use json5 ext and add some comments

Reply:
see https://github.com/web-infra-dev/rsbuild/blob/main/.github/renovate.json5

Reply:
json5 is better, I will change to json5

Reply:
changed to json5 now